.circleci/config.yml

```diff
@@ -554,7 +554,7 @@ jobs:
           name: Setup Database
           command: |
             bundle exec rails db:create --trace
-            bundle exec rails db:structure:load --trace
```

Comment:
This will need to be changed in the `build` job when we make Rails 7.0 the only version, as `rails db:structure:load` was deprecated and Removed in Rails 7.0.